### ISSUE TITLE
[Bug]: Missing Files

### ISSUE THREAD

**Issue by jdietz02** [advplyr/audiobookshelf#1846]
### Describe the issue

It keeps telling me I have books files missing after a scan, but the file is still there

### Steps to reproduce the issue

1. Run a scan then it show missing file. i can select the file and re scan that file and the warning goes away untill the next scan then it show the warning again

### Audiobookshelf version

V2.2.23

### How are you running audiobookshelf?

Docker

**Comment by jdietz02** [comment]
So you are saying it is an issue with my setup, but if that were the case wouldn't all 351 books show missing not just the 25 books that are showing missing?

**Comment by CLHatch** [comment]
> So you are saying it is an issue with my setup, but if that were the case wouldn't all 351 books show missing not just the 25 books that are showing missing?

No, he saying it's probably an issue with your setup, and to come chat about it to be sure.  I'm betting that those 25 files are in the wrong place or something.
